### PR TITLE
[WIP] merge proxy command to serve command

### PR DESCRIPTION
**Pull request by ramilexe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,32 @@ The currently supported standard endpoints are:
 
 TODO: Add the rest of the standard endpoints and unique endpoints (e.g. getSlice)
 
+
```
Comment (i-norden) [inline README.md:124]: Very nice table, I think this helps a lot

File: cmd/serve.go [hunk 2]
```diff
@@ -16,8 +16,13 @@
 package cmd
 
 import (
+	"errors"
+	"github.com/vulcanize/gap-filler/pkg/mux"
```
Comment (i-norden) [inline cmd/serve.go:20]: Really minor but can we move this external dep out of the builtins

**Review by i-norden — APPROVED** [review]
The table with all the params and their env variables is a great addition. Only one small reformatting fix I caught, and we should also bump the version. Otherwise LGTM!

**Comment by i-norden** [comment]
Actually we can probably wait to bump version til all the upcoming changes are in

File: cmd/serve.go (hunk 2 above)
Comment (arijitAD) [reply cmd/serve.go]: Done

**Review by i-norden — APPROVED** [review]
I think this LGTM. It's a bit hard to review since all the changes from the integration test branch are here now. In the future we should rebase ontop of master instead of merging master back into the working branch.

**Pull request merged**